library/core/src/fmt/mod.rs

```diff
@@ -112,9 +112,9 @@ pub trait Write {
     ///
     /// # Errors
     ///
-    /// This function will return an instance of [`Error`] on error.
+    /// This function will return an instance of [`std::fmt::Error`] on error.
```

Comment:
This should unbreak the doc build.

```suggestion
    /// This function will return an instance of [`std::fmt::Error`][Error] on error.
```